src/main/java/com/redhat/devtools/lsp4ij/features/codeAction/intention/LSPIntentionCodeActionSupport.java

```diff
@@ -99,7 +99,7 @@ private static CompletableFuture<List<CodeActionData>> getCodeActionsFor(@NotNul
                         return Collections.emptyList();
                     }
                     return codeActions.stream()
-                            .filter(Objects::nonNull)
+                            .filter(LSPIntentionCodeActionSupport::isInvalidValidCodeAction)

```

Comment:
so we're only returning CodeActionData for invalid code actions? Seems counter intuitive